src/routes/chains/chains.controller.spec.ts

```diff
@@ -402,4 +402,60 @@ describe('Chains Controller (Unit)', () => {
         });
     });
   });
+
+  describe('GET /:chainId/about', () => {
+    it('Success', async () => {
+      const chainId = faker.random.numeric();
```

Comment:
nitpick: you can get the `chainId` from the `chainBuilder()` itself. This can be useful if you don't want to set any chain id specifically as it generates one for you.

Reply:
True, thanks! Changed in [ad8a8a3](https://github.com/5afe/safe-client-gateway-nest/pull/240/commits/ad8a8a37a0635cfebfe73331756645bab851417c)